manifests/charts/gateways/istio-egress/templates/poddisruptionbudget.yaml

```diff
@@ -1,5 +1,5 @@
-{{- if .Values.global.defaultPodDisruptionBudget.enabled }}
 {{ $gateway := index .Values "gateways" "istio-egressgateway" }}
+{{- if and $gateway.autoscaleEnabled .Values.global.defaultPodDisruptionBudget.enabled (ge $gateway.autoscaleMin 2.0) }}
```

Comment:
> (ge $gateway.autoscaleMin 2.0)

This comparison is wired. It should have work with `int` or at least typecast to `int`
https://stackoverflow.com/questions/46009933/kubernetes-helm-chart-if-condition-check?rq=1

Reply:
Or maybe check `autoscaleMin >= 2` during [translateHPASpec](https://github.com/istio/istio/blob/master/operator/pkg/translate/translate_value.go#L397)